### PR TITLE
[INJIMOB-748] update activity log messages

### PR DESCRIPTION
**Pull request by vharsh**
# Other minor details

1. Move ActivityLogType to the ActivityLog component.
2. Update Activitylog for all languages as per ticket requirements.

Link: Go [here](https://mosip.atlassian.net/browse/INJIMOB-748) for screenshots.

### CODE REVIEW

File: locales/kan.json [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+    "VC_RECEIVED_NOT_SAVED": "{{idType}} {{id}} ಉಳಿಸಲು ಸಾಧ್ಯವಾಗಲಿಲ್ಲ.",
+    "VC_DELETED": "{{idType}} {{id}} ಯಶಸ್ವಿಯಾಗಿ ಅಳಿಸಲಾಗಿದೆ.",
+    "VC_DOWNLOADED": "{{idType}} {{id}} ಡೌನ್‌ಲೋಡ್ ಮಾಡಲಾಗಿದೆ.",
+    "VC_REVOKED": "{{idType}} {{id}} ನೀಡುಗರಿಂದ ರದ್ದುಗೊಳಿಸಲಾಗಿದೆ.",
+    "VC_SHARED_WITH_VERIFICATION_CONSENT": "ಹಾಜರಿ ಪರಿಶೀಲನೆಗಾಗಿ {{idType}} {{id}} ಅನುಮತಿಯೊಂದಿಗೆ ಹಂಚಲಾಗಿದೆ.",
+    "VC_RECEIVED_WITH_PRESENCE_VERIFIED": "{{idType}} {{id}} ಸ್ವೀಕರಿಸಲಾಗಿದೆ ಮತ್ತು ಹಾಜರಿ ಪರಿಶೀಲನೆ ಯಶಸ್ವಿಯಾಗಿದೆ.",
+    "VC_RECEIVED_BUT_PRESENCE_VERIFICATION_FAILED": "{{idType}} {{id}} ಸ್ವೀಕರಿಸಲಾಗಿದೆ ಆದರೆ ಹಾಜರಿ ಪರಿಶೀಲನೆ ವಿಫಲವಾಗಿದೆ.",
+    "PRESENCE_VERIFIED_AND_VC_SHARED": "ಮುಖ ಪರಿಶೀಲನೆ ಯಶಸ್ವಿಯಾಗಿದೆ ಮತ್ತು {{idType}} {{id}} ಹಂಚಲಾಗಿದೆ.",
+    "PRESENCE_VERIFICATION_FAILED": "{{idType}} {{id}} ಹಂಚಿಕೊಳ್ಳಲು ಪ್ರಯತ್ನಿಸುವಾಗ ಮುಖ ಪರಿಶೀಲನೆ ವಿಫಲವಾಗಿದೆ.",
+    "QRLOGIN_SUCCESFULL": "{{idType}} {{id}} ಬಳಸಿ QRLogin ಯಶಸ್ವಿಯಾಗಿದೆ.",
```
Comment (vharsh) [inline locales/kan.json:14]:
```suggestion
    "QRLOGIN_SUCCESFULL": "{{idType}} {{id}} ಬಳಸಿ QRಲಾಗಿನ್ ಯಶಸ್ವಿಯಾಗಿದೆ.",
```

File: shared/openId4VCI/Utils.ts [hunk 2]
```diff
@@ -23,6 +23,17 @@ export const Issuers = {
   ESignet: 'ESignet',
 };
 
+/**
+ * @param issuer the issuer of the VC as per the VC metadata in MMKV
+ * @returns the ID-type to be used for further translation
+ */
+export function getIdType(issuer: string | undefined): string {
```
Comment (vijay151096) [inline shared/openId4VCI/Utils.ts:30]: why do we need to explicitly mention NATIONAL_ID and INSURANCE_ID, can't it be handled using 
ID_TYPE

File: locales/ara.json [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+    "VC_RECEIVED_WITH_PRESENCE_VERIFIED": "تم استلام {{idType}} {{id}} ونجح التحقق من الحضور.",
+    "VC_RECEIVED_BUT_PRESENCE_VERIFICATION_FAILED": "تم استلام {{idType}} {{id}} ولكن فشل التحقق من الحضور.",
+    "PRESENCE_VERIFIED_AND_VC_SHARED": "نجح التحقق من الوجه وتم مشاركة {{idType}} {{id}}.",
+    "PRESENCE_VERIFICATION_FAILED": "فشل التحقق من الوجه أثناء محاولة مشاركة {{idType}} {{id}}.",
+    "QRLOGIN_SUCCESFULL": "نجح تسجيل الدخول باستخدام QR باستخدام {{idType}} {{id}}.",
+    "WALLET_BINDING_SUCCESSFULL": "تم تفعيل {{idType}} {{id}} بنجاح.",
+    "WALLET_BINDING_FAILURE": "فشل تفعيل {{idType}} {{id}}.",
+    "VC_REMOVED": "تم إزالة {{idType}} {{id}} من المحفظة.",
+    "TAMPERED_VC_REMOVED": "تم إزالة {{idType}} {{id}} من المحفظة بسبب العبث.",
+    "NATIONAL_ID": "الهوية الوطنية",
```
Comment (vijay151096) [inline locales/ara.json:19]: If not needed, can we remove the NATIONAL_ID and INSURANCE_ID

Comment (vharsh) [reply locales/ara.json:19]: This is needed for the story to show the type of VC, this is temporary as later there'd be a better way to tell apart ID-Type when more IDs get supported by Inji.

Comment (vharsh) [reply locales/ara.json:19]: E.g. 

`National ID 1234 downloaded` --> downloaded from E-Signet using OIDC flow / OTP flow
`Insurance Card 1234 downloaded` --> downloaded from OIDC impl at Sunbird side

File: shared/openId4VCI/Utils.ts (hunk 2 above)
Comment (vharsh) [reply shared/openId4VCI/Utils.ts:30]: This is a temporary change, I should document it likewise.
Right now I'm using this field to figure out the kind of card it is.

Comment (vharsh) [reply shared/openId4VCI/Utils.ts:30]: Got it, you want me to use this?

```typescript
export const ID_TYPE = {
  MOSIPVerifiableCredential: i18n.t('VcDetails:nationalCard'),
  InsuranceCredential: i18n.t('VcDetails:insuranceCard'),
};
```

Comment (vharsh) [reply shared/openId4VCI/Utils.ts:32]: I've implemented this.

**Pull request merged**